gnomad_constraint/utils/constraint.py

```diff
@@ -396,6 +406,183 @@ def apply_models(
     return ht.annotate(obs_exp=ht.observed_variants / ht.expected_variants)
 
 
+def calculate_mu_by_downsampling(
+    genome_ht: hl.Table,
+    context_ht: hl.Table,
+    recalculate_all_possible_summary: bool = True,
+    omit_methylation: bool = False,
+    count_singletons: bool = False,
+    keep_annotations: Tuple[str] = (
+        "context",
+        "ref",
+        "alt",
+        "methylation_level",
+    ),
+    ac_cutoff: int = 5,
+    downsampling_level: int = 1000,
+    total_mu: float = 1.2e-08,
+    pops: Tuple[str] = (),
+    min_cov: int = 15,
+    max_cov: int = 60,
+    gerp_lower_cutoff: float = -3.9885,
+    gerp_upper_cutoff: float = 2.6607,
+) -> hl.Table:
+    """
+    Calculate mutation rate using the downsampling with size specified by `downsampling_level` in genome sites Table.
+
+    Prior to computing mutation rate the only following variants are kept:
+        - variants with the mean coverage in the gnomAD genomes between `min_cov` and `max_cov`.
+        - variants where the most severe consequence was 'intron_variant' or
+            'intergenic_variant'.
+        - variants with the GERP score between `gerp_lower_cutoff` and `gerp_upper_cutoff` (these default to -3.9885 and
+        2.6607, respectively - these values were precalculated on the GRCh37 context
+        Table and define the 5th and 95th percentiles).
+        - high-quality variants: `exome_ht.pass_filters`.
+        - variants with allele count below `ac_cutoff`: `(freq_expr.AC <= ac_cutoff)`.
```

Comment:
just some reformatting:

```suggestion
        - variants with the mean coverage in the gnomAD genomes between `min_cov` and 
          `max_cov`.
        - variants where the most severe consequence was 'intron_variant' or
          'intergenic_variant'.
        - variants with the GERP score between `gerp_lower_cutoff` and 
          `gerp_upper_cutoff` (these default to -3.9885 and 2.6607, respectively - 
          these values were precalculated on the GRCh37 context Table and define the 
          5th and 95th percentiles).
        - high-quality variants: `exome_ht.pass_filters`.
        - variants with allele count below `ac_cutoff`: `(freq_expr.AC <= ac_cutoff)`.
```